osquery/logger/plugins/tls_logger.h

```diff
@@ -60,13 +60,20 @@ class TLSLoggerPlugin : public LoggerPlugin {
     return true;
   }
 
+  bool usesLogEvent() override {
+    return true;
```

Comment:
If this feature is not desired, does this default-true cause additional calling/no-op overhead? Could this value be determined by the value of the flag?
```
return !FLAGS_logger_tls_event_types.empty();
```